### DISCUSSION TITLE
Not sticking to build plate

### DISCUSSION THREAD

**Discussion by jpdore15** [sn4k3/UVtools#774] [Q&A]
Saturn 3 ultra with elegoo water washable resin. Bottom exposure= 25/30, normal=2.5/3. 

Any help would be greatly appreciated. I am very new to the 3d printing game, I have printed a few of the calibration tests successfully but, when trying to print the exposure finder the print is not sticking to the build plate at all. I have tried to print twice and both times it did the same thing. When I printed the validation matrix sliced in voxedance at 25s/2.5s it stays on the build plate. 

Thank you.

**Comment by sn4k3** [comment]
If other prints stick to build plate the problem should be the test you run from UVtools, some users are encountering that problem on new CTB format. However, the initial base layers must stick to plate, if not it could be another problem.

**Reply by jpdore15** [reply]
Thank you for your response. Other prints are sticking to the build plate at 25s base layer and 2.5 sec normal. Do I need to change the file format?

**Reply by sn4k3** [reply]
You can't change file format, printers often only supports one file format, so you need to stick with it.

**Reply by jpdore15** [reply]
Ok, thank you! Is there anything you would suggest trying to get it to stick correctly?

**Reply by sn4k3** [reply]
If nothing stick to plate, try to increase bottom exposure to 50s just to be sure.
If first layers stick but then others not you need to discard UVtools test and go old method: 1 print/test at the time

**Comment by Hazoooka** [comment]
Is this printer newly bought or an old one?

The problem with adhesion can be also due to the dust inside the machine that builds up over time.
Some, if not all the printers use mirrors for their UV LEDs, I just know my Sonic Mini.
The mirror gets pretty dusty, as the rest of the parts inside I presume.
Cleaning the mirror on the old machine can do the trick.

For comparison, me and my friend have the same printer model.
I get a proper adhesion with 6s bottom layers exposure while my friend needs 35s and still he says his models do not stick that hard.

**Reply by jpdore15** [reply]
New printer. I have been using voxeldance, and just switched to chitubox. I have been able to print minis and single test prints using both slicers. I have not yet tried using the file from chitubox or changing the bottom exposure to 50s.
